### ISSUE TITLE
This project needs a clean-up

### ISSUE THREAD

**Issue by t1ra** [sundowndev/hacker-roadmap#24]
**Is your feature request related to a problem? Please describe.**
Just by going through 3-5 random repos listed here, I've found some awful, awful code that absolutely don't meet the CONTRIBUTING.md rule of
> Please don't submit unverified or untested tools. We only want to share tools that have good community of contributors. **If you've created a tool, be sure the code is stable and will be maintained.**

(emphasis mine)

**Describe the solution you'd like**
- A higher standard to hold repo code quality to, or
- A sort though currently listed repos and removing poor quality repos.

**Additional context**
I've found multiple issues looking through only a fraction of the repos listed. The worst offender is probably https://github.com/Manisso/Crips. Here are a few of the problems with it, including in the *opening lines*
```python
d3=os.system("curl http://ipinfo.io/ip")
os.system("clear && clear && clear")
```
- I assume this call is to get the IP address of the user? Not only is this outside of the scope of the tool and otherwise unnecessary, there much better ways to use it than shelling out to a platform-dependent program accessing a random host via http.
- As stated previously, not only is `curl` effectively exclusively available on UNIX/UNIX-like platforms, it doesn't, as far as i'm aware, come with macOS, limiting this script to *most* Linux distros. On top of that, this isn't the only call to curl, and clearly not the only use of `os.system`. But of course, the creator still gives install instructions for Windows, never mentioning the curl dependency. `curl` is generally used throughout the file, rather than say, `request`.
- `os.system("clear && clear && clear")`. Seriously? I get that they're trying to make an epikx hax0r tool but **please, please leave clearing the screen up to the user**. And again, the use of `clear` through a fucking shell call restricts this to UNIX/UNIX-like systems.
And that's just in the first two lines.
- Its install and update scripts are written in Bash, rather than being remotely competent and using `pip`, the tool *packaged with Python* for *this EXACT job*.
- Oh and of course its filled with gamer ascii art and "greetz" (with facebook links!) to the "fsociety team", including in the `update.sh` script that not only spams `clear` but uses `rm -rf` on a directory created by the user. Jesus. Fucking. Christ.

----

Reading through some of this code makes your intro to your repo incredibly ironic:
> Read books, manuals, articles, be curious and not just a script kiddie.
Because it seems like half of these tools were written by them.

If you're interested in cleaning up the repo, I'd gladly go through and remove blatantly poorly written code. Regardless, this project is on the trending for today section on GitHub, please consider having a higher barrier for entry.

**Comment by sundowndev** [comment]
Hi, thank you for pointing this out. I still agree about that contribution rule "project must be tested and maintained". Unfortunately, I don't have time to test every single tool I come across. I don't even remember where I found that Crips repository. But I know that same guy created fsociety, a pen testing framework with a lot of tools (which in fact remain on a `fsociety.py` file with 2k lines of code...that just clones other repositories). I guess these projects were just recommended to me and I added it to my curated list. Anyway, I'll take a look. Feel free to submit a PR about it!
